plugins/modules/foreman_ptable.py

```diff
@@ -320,11 +321,11 @@ def main():
 
     if affects_multiple:
         entities = module.list_resource('ptables')
-        if not module.desired_absent:  # not 'thin'
-            entities = [module.show_resource('ptables', entity['id']) for entity in entities]
         if not entities:
             # Nothing to do; shortcut to exit
             module.exit_json(changed=False)
+        if not module.desired_absent:  # not 'thin'
```

Comment:
This does not really belong in here ;-)

Reply:
It came along as a minor optimisation.
And i have two other PR's open that grew a lot more out their original scope. %)